src/engine/ruleset/decoders/f5/f5-apm-own.yml

```diff
@@ -0,0 +1,451 @@
+---
+name: decoder/f5-apm-own/0
+
+metadata:
+  module: F5
+  title: F5 APM Decoder for own providers
+  description: Decodes HEADER F5 BIG-IP Access Policy Manager logs of own providers.
+  compatibility: >
+    New Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - TBD
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~payload>
+
+parse:
+  logpar:
+    - ~payload: <~>[<process.pid>]<~/ignore/:> <event.provider>:<~/ignore/ ><~content>
```

Comment:
Improve (Design): If is possible, you could use a single temporary variable and in the end delete only this one. For example: ~tmp